Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -395,355 +395,677 @@ public enum Language
     {
 #pragma warning disable 1591
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Abap")]
+        [Parameter(Value = "abap")]
         Abap,
+
         [Parameter(Value = "ActionScript")]
         ActionScript,
+        
+        [Parameter(Value = "ada")]
         Ada,
+        
+        [Parameter(Value = "apex")]
         Apex,
+        
         [Parameter(Value = "AppleScript")]
```

Comment:
Interesting the case doesn't matter for the request... It does seem more correct though to match the API response (and if you look at the language element of the repo returned in your example above the "official" casing is AppleScript 

Perhaps an integration test that loops through every language enum and does the search and verifies the returned language (eg repository.language in the case of repo search)